pre_install_report/README.md

```diff
@@ -58,6 +58,10 @@ $ export INGRESS_HOST=externalIP=$(kubectl -n <ingress-namespace> get service <n
 $ export INGRESS_HTTP_PORT=$(kubectl -n <ingress-namespace> get service <nginx-ingress-controller-name> -o jsonpath='{.spec.ports[?(@.name=="http")].port}')
 $ export INGRESS_HTTPS_PORT=$(kubectl -n <ingress-namespace> get service <nginx-ingress-controller-name> -o jsonpath='{.spec.ports[?(@.name=="https")].port}')
 ```
+The command to determine the Ingress Host may be slightly different with Amazon EKS:
```

Comment:
Expand "EKS" to "Elastic Kubernetes Service (EKS)" assuming this is the first use of the term.

Reply:
Done

Reply:
I'm not seeing where this is comment is actually satisfied in the commits.

Reply:
Will Push correctly now.

Reply:
Done. Pushed again.